pkg/api/dtos/frontend_settings.go

```diff
@@ -64,12 +65,18 @@ type FrontendSettingsLicenseInfoDTO struct {
 	AppUrl      *string `json:"appUrl,omitempty"`
 }
 
+type FrontendSettingsAzureCloudDTO struct {
```

Comment:
This can be removed now too

Reply:
Oh, dang - I thought I had removed that.  Thanks for catching that.